Rebuilded logic for paging

giancarlobi opened the pull request:
As for Presentation API 2.1 (https://iiif.io/api/presentation/2.1/#paging) paging requires json output more stricted so the new logic use 3 kind of object: Collection without paging, Collection paging parent and Collection paging page.

giancarlobi committed:
Rebuilded logic for paging

As for Presentation API 2.1 (https://iiif.io/api/presentation/2.1/#paging) paging requires json output more stricted so the new logic use 3 kind of object: Collection without paging, Collection paging parent and Collection paging page.

DiegoPino commented:
First, why are you awake doing this 🤣 Second nice work =)

giancarlobi commented:
It was a very very hard job ...

giancarlobi commented:
and  are you sure I'm awake and not coding while sleeping ?!?!??!

DiegoPino commented:
jajajajaja. You are probably awake i know you! You never stop until done =)

giancarlobi commented:
You are right, tengo la cabeza muy muy dura ... and now time to sleep!

giancarlobi commented:
Have a nice weekend amigo !

DiegoPino commented:
Tu tb!

El El vie, 22 de mar. de 2019 a las 21:14, Giancarlo <
notifications@github.com> escribió:

> Have a nice weekend amigo !
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/mnylc/islandora_iiif/pull/6#issuecomment-475826848>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGn854xFrKMy4zyZDIToRmQR_mx95E3Aks5vZYAAgaJpZM4cEoPq>
> .
>
-- 
Diego Pino Navarro
Digital Repositories Developer
Metropolitan New York Library Council (METRO)

giancarlobi committed:
Use info.json when no children

Manifest of image with this code now return correct width/height of image as width/height are not store into RELS_INT when largeImage CMODEL.

giancarlobi committed:
type error

giancarlobi committed:
Changed thumbnail width to 100px

giancarlobi commented:
@DiegoPino I think 100px is better for thumbnail, i.e. into info tab of BookReader of Archipelago

DiegoPino commented:
I agree

El El jue, 11 de abr. de 2019 a las 08:05, Giancarlo <
notifications@github.com> escribió:

> @DiegoPino <https://github.com/DiegoPino> I think 100px is better for
> thumbnail, i.e. into info tab of BookReader
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/mnylc/islandora_iiif/pull/6#issuecomment-482087025>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGn853jNk7i0p-kSggCwgImt7B6bptR6ks5vfyUdgaJpZM4cEoPq>
> .
>
-- 
Diego Pino Navarro
Digital Repositories Developer
Metropolitan New York Library Council (METRO)

DiegoPino reviewed (approved):
A lot of new thing!!

DiegoPino commented:
@giancarlobi let me know if you are ready to get this merged or you are planning on adding more commits. Thanks!

giancarlobi commented:
@DiegoPino no more commits for now. I tested also how islandora solr metadata are added to manifest (your code), we don't need Islandora Solr Metadata  selected (here admin/islandora/metadata) to make it working, great!

DiegoPino commented:
@giancarlobi i think its time to let other people know about your amazing work here! A lot of islandora users in the groups will benefit of your code. 👏

giancarlobi commented:
@DiegoPino I left this to you, for me ok.

DiegoPino commented:
Oh, its your code. Do it whenever you feel its ok (nor not). I won't take credit for this!!

giancarlobi commented:
It's OUR code so ... feel you free to make it public to community as you consider it better.

giancarlobi commented:
@DiegoPino I think you can explain better than me what this module does and why it is so important.
